pkg/api/client/proxy.go

```diff
@@ -159,6 +159,19 @@ func (c ProxyScriptsAPI) ListScripts(namespace string) (scripts kubtest.Scripts,
 	return c.getScriptsFromResponse(resp)
 }
 
+// GetExecutions list all executions in given script
+func (c ProxyScriptsAPI) AbortExecution(scriptID, id string) error {
+	uri := fmt.Sprintf("v1/scripts/%s/executions/%s/abort", scriptID, id)
```

Comment:
Same as previous. Can we have it like a const? and v1 here should not be hardcoded.